test/samlp.logout.custom_store.tests.js

```diff
@@ -241,6 +250,7 @@ describe('samlp logout with Session Participants - Custom Provider', function ()
       it('should validate LogoutRequest to Session Participant', function () {
         expect(sessionParticipantLogoutRequest).to.exist;
         expect(xmlhelper.getIssueInstant(sessionParticipantLogoutRequest)).to.exist;
+        expect(new Date(xmlhelper.getIssueInstant(sessionParticipantLogoutRequest)).getTime()).to.equal(frozenTime.getTime());
```

Comment:
Minor but I guess this assertion (once you remove the `new Date` part) makes the `.to.exist` assertion redundant